scripts/get_visualization.py

```diff
@@ -0,0 +1,84 @@
+"""Creates a collection of graphs summarizing results saved to a directory."""
+import argparse
+from typing import Dict, Tuple
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+CUSTOM_APPROACHES = ["llm-standard-plan", "llm-multi", "llm-multi-plan"]
+BASELINE_APPROACHES = ["fd", "pyperplan"]
+
+
+def _main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--input_path", type=str)
+    parser.add_argument("--output_path", type=str)
+    args = parser.parse_args()
+    custom_approaches, baseline_approaches = _construct_dictionaries(
+        args.input_path)
+    _get_visualization(custom_approaches, baseline_approaches,
+                       args.output_path)
+
+
+def _construct_dictionaries(input_path: str) -> Tuple[Dict, Dict]:
+    custom_approaches: Dict = {}
```

Comment:
make these type annotations more specific, like you do later on. do this here, and also in the output annotation for this function